cedar-policy/src/api.rs

```diff
@@ -1709,6 +1726,14 @@ impl PolicySet {
         Ok(())
     }
 
+    /// Remove a `Template` from the `PolicySet`
+    pub fn remove_template(&mut self, template_id: PolicyId) -> Result<(), PolicySetError> {
```

Comment:
We should be explicit in the documentation about what happens if you remove a template that has active links?

Reply:
I decided that should be an error.